src/engraving/dom/measurenumber.cpp

```diff
@@ -61,6 +59,16 @@ MeasureNumber::MeasureNumber(const MeasureNumber& other)
     initElementStyle(&measureNumberStyle);
 }
 
+PropertyValue MeasureNumber::getProperty(Pid id) const
+{
+    switch (id) {
+    case Pid::HPLACEMENT:
+        return style().styleV(Sid::measureNumberHPlacement);
+    default:
+        return MeasureNumberBase::propertyDefault(id);
```

Comment:
`getProperty` vs `propertyDefault`... is that intentional?

If it's intentional that all properties always return their default value... should they be properties at all?